build.gradle

```diff
@@ -25,8 +25,8 @@ gradlePlugin {
         jsonschema2dataclassPlugin {
             id = 'org.jsonschema2dataclass'
             version = getTag()
-            implementationClass = 'com.github.eirnym.js2p.JsonSchemaPlugin'
-            displayName = 'Extended Gradle JsonSchema2Pojo Plugin'
+            implementationClass = 'com.github.js2d.JsonSchemaPlugin'
+            displayName = 'Extended Gradle JsonSchema2DataClass Plugin'
```

Comment:
Revert displayName

Reply:
I did it, but why this old name ?

Reply:
_Currently_ this is a gradle plugin for JsonSchema2Pojo library. Probably this should be stated better.